src/map/battle.c

```diff
@@ -3025,16 +3025,6 @@ static int battle_calc_skillratio(int attack_type, struct block_list *src, struc
 			}
 			//Skill damage modifiers that stack linearly
 			if(sc && skill_id != PA_SACRIFICE){
-#ifdef RENEWAL_EDP
-				if( sc->data[SC_EDP] ){
-					if( skill_id == GC_COUNTERSLASH ||
-#ifndef RENEWAL
```

Comment:
This looks confusing, but it is because I updated sonic blow in Sonic Blows' rebalance ( e94052e ), but this is a renewal change, so removing it is correct.